docs/guides/cache/azblob.md

```diff
@@ -1,62 +1,46 @@
 # Azure Blob Storage cache storage
 
-> **Warning**
->
-> The `azblob` cache is currently unreleased. You can use it today, by using
-> the `moby/buildkit:master` image in your Buildx driver.
+> This cache backend is unreleased. You can use it today, by using the
+> `moby/buildkit:master` image in your Buildx driver. {: .warning }
 
 The `azblob` cache store uploads your resulting build cache to
 [Azure's blob storage service](https://azure.microsoft.com/en-us/services/storage/blobs/).
 
 > **Note**
 >
-> The `azblob` cache storage backend requires using a different driver than
-> the default `docker` driver - see more information on selecting a driver
-> [here](../drivers/index.md). To create a new docker-container driver (which
-> can act as a simple drop-in replacement):
+> This cache storage backend requires using a different driver than the default
+> `docker` driver - see more information on selecting a driver
+> [here](../drivers/index.md). To create a new driver (which can act as a simple
+> drop-in replacement):
 >
 > ```console
 > docker buildx create --use --driver=docker-container
 > ```
 
-To import and export your cache using the `azblob` storage backend we use the
-`--cache-to` and `--cache-from` flags and point it to our desired blob using
-the required `account_url` and `name` parameters:
+## Synopsis
 
 ```console
-$ docker buildx build --push -t <user>/<image> \
-  --cache-to type=azblob,account_url=https://myaccount.blob.core.windows.net,name=my_image \
-  --cache-from type=azblob,account_url=https://myaccount.blob.core.windows.net,name=my_image
+$ docker buildx build . --push -t <registry>/<image> \
```

Comment:
This LGTM, I'm happy to have this be the standard way of writing this :tada:

We can make sure to copy it through to the rest of the pages as well later.

Reply:
ACK